src/common/plugin.ts

```diff
@@ -47,10 +55,54 @@ export abstract class Plugin implements IPlugin {
 
     const response = this.container.channelService.hasPermission(channelName, this.permission);
     if (!response) {
-      message.reply(`Please use this command in a \`${this.permission}\` channel`);
+      const rooms = Object.values(Constants.Channels[this.permission]).splice(0, 3);
```

Comment:
If the permission is `Public` this will return the first 3 public channels in Constants.ts, which are `Announcements`, `Welcome`, and `ClassInvite`. None of which the user can type in

Reply:
Yeah I noticed that was debating about reordering the list so that it reflects what info we need better, or making a custom public like what I did with private.  Another idea I just came up with while responding is making a new channel type for channels like this, although I think that should be a superate pull request. @c650 @tanndlin Thoughts on adding a seperate channel type for these announcement type channels?

Reply:
We could have an `info` channel type for those